src/controllers/reactWebviewBaseController.ts

```diff
@@ -126,9 +133,20 @@ export abstract class ReactWebviewBaseController<State, Reducers>
      */
     constructor(
         protected _context: vscode.ExtensionContext,
+        protected vscodeWrapper: VscodeWrapper,
         private _sourceFile: string,
         private _initialData: State,
-    ) {}
+    ) {
+        if (!vscodeWrapper) {
+            vscodeWrapper = new VscodeWrapper();
+        }
+
+        if (!ReactWebviewBaseController._channel) {
+            ReactWebviewBaseController._channel =
+                vscodeWrapper.createOutputChannel("MSSQL WebViews"); // TODO: use name of webview?
```

Comment:
Do we need to have a different output channel for webview? 

I feel it will be easy to debug if we can see all the logs in one output channel. Otherwise we will have to keep switching between them if we are debugging events that are across extension and webviews

Maybe we can add a prefix and colors to differentiate between event sources.

Reply:
Good call.  I'll reuse that output channel and use prefixes to differentiate between webviews.